### ISSUE TITLE
documentation should contain a hint for to long paths for ftp upload

### ISSUE THREAD

**Issue by rallek** [zikula/core#2277]
I just run into a problem with ftp upload using filezilla.

There have been 21 files which could not be uploaded. Retrying also do not the expected. After some analysing I found out the path of some files are to long. This is in my case the Acme demo bundle (whatever this is). Here one example of one file path:
````
D:\Users\Ralf\Zikula\ci.zikula.org\Zikula_Core-1.4.0.build450.tar\Zikula_Core-1.4.0.build450\Zikula_Core-1.4.0\vendor\sensio\distribution-bundle\Sensio\Bundle\DistributionBundle\Resources\skeleton\acme-demo-bundle\Acme\DemoBundle\Controller\SecuredController.php
````
When I moved the DemoBundle/... to another place the upload went fine.

We should make a hint in the documentation to take care not to have the own unziped files in a deep folder structure already.

I am running Win7/64bit and Filezilla 3.10.1.1 but I am shure there are other ftp tools with the same behavior.

**Comment by craigh** [comment]
the documentation already lists the solution to this problem:

> upload the entire archive (.zip or .tgz file) to your server and then unzip them there

**Comment by rallek** [comment]
sorry, I totally disagree! Only very few shared hosting are offering a command line! Most have to upload via ftp (that is my test scenario at the moment). 

And if you have access to a comand line you are running into trouble of file ownership as I did. 

Please reopen and lable it as documentation. You can assign me and put it to 1.4.1.

**Comment by Guite** [comment]
The problem is probably related to Windows. Windows has a maximum path length of 260 characters.

This is not a Zikula-specific issue though.

**Comment by Portugao** [comment]
It is also possible, that this is an issue with fileZilla.
With fire ftp there are no problems with that for me.

**Comment by rallek** [comment]
thanks for reopening. I will think about a nice sentence for the documentation.

I just want to avoid support issues

**Comment by craigh** [comment]
doesn't filezilla have a 'decompress file' command? I think it does.

documentation doesn't avoid support issues.

**Comment by craigh** [comment]
unzipping from within filezilla:

http://smallbusiness.chron.com/extract-zip-file-filezilla-57084.html

**Comment by rallek** [comment]
unzipping at the server is not possible. You can open a compressed file via ftp locally with this trick

**Comment by rallek** [comment]
proposal is given in 78c5ef2

Mark checked my english ;-)

**Comment by craigh** [comment]
> unzipping at the server is not possible.

apparently I was wrong. Sorry.

I must be thinking of cpanel or other FTP clients I used to use.

I'm still unsure about if the additional documentation will help or not.

**Comment by craigh** [comment]
closed by #2278
